src/frontend/screenshot_window.rs

```diff
@@ -32,18 +33,27 @@ pub struct ScreenshotWindowModel {
     toolbar: Controller<Toolbar>,
 }
 
+#[derive(Debug)]
+pub enum MouseEvent {
+    MouseMove(f64, f64),
+    MosePress(i32, f64, f64),
+    MouseRelease(i32, f64, f64),
+}
```

Comment:
what should `i32` represent?

Maybe you could apply my suggestion with the `widgets/` directory and some type-abstractions so we have something like:
```rust
#[derive(Debug)]
pub enum MouseEvent {
    MouseMove(Point),
    MosePress {
        monitor_id: i32, // is that right?
        point: Point,
    }
    MouseRelease {
        monitor_id: i32, // is that right?
        point: Point,
    }
}
```

Reply:
It represents the pressed/released mouse button. A dedicated type might be overkill but I can change the variants to struct variants for increased readability if you like.

Reply:
done